PlayFabSDK/source/Qos/RegionPinger.cs

```diff
@@ -93,15 +153,12 @@ private async Task<int> PingInternalAsync()
 
                     if (result < int.MaxValue)
                     {
-                        return (int)result;
+                        return (int) result;
```

Comment:
FYI, the spacing standards for the SDKs are "press Ctrl+K+D" in Visual Studio.
This makes it very fast and easy to apply, and reduces ambiguity.
(Admittedly, VS 17 & 19 produce very slight differences, but either will be accepted, since it's sufficiently similar and essentially no manual work)

Reply:
Sorry wasn't an intentional change. I use rider and it did this for me. Feel free to revert.

You could consider using an [.editorconfig](https://editorconfig.org/) to get consistent behavior across editors (though even then there are some implementation differences.